.travis.yml

```diff
@@ -35,7 +35,8 @@ install:
 
 before_script:
   - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/cuda/lib64:$CONDA_DIR/lib
-  - export PATH=$CONDA_DIR/bin:$PATH
+  - export PATH=$CONDA_DIR/bin:/home/travis/build/yahoo/CaffeOnSpark/spark-1.6.0-bin-hadoop2.6/bin:$PATH
+  - sudo apt-get install libboost-all-dev
```

Comment:
This statement should be removed. caffe-public should have installed boost already.